pytripgui/field_vc/field_cont.py

```diff
@@ -47,10 +54,33 @@ def set_model_from_view(self):
         else:
             self.model.isocenter = []
 
-        self.model.gantry = self.view.gantry_angle
-        self.model.couch = self.view.couch_angle
+        self.model.display_angles_in_standard = self.view.angles_standard
+        self.model.gantry, self.model.couch = self._get_view_angles_in_trip_standard()
```

Comment:
Explain clearly in comment that all angles are stored in model standard in the TRiP98 standard.
I'd even consider encoding this in the variable name:
```
self.model.gantry_angle_trip98 = ...
```

In this way whenever a programmer access `self.model.gantry_angle_trip98` there will be no doubt in which format the angle is specified. This will be clear from the variable name